docs/reset-techpass-mfa-for-new-device.md

```diff
@@ -1,40 +1,58 @@
 # Reset TechPass MFA
 
-This article guides how to reset TechPass MFA.
+This article guides how TechPass users can reset TechPass MFA.
 
-**Audience**
+## Audience
 
-- Users who have a TechPass account and domain is ```techpass.gov.sg```.
+TechPass users may want to reset the MFA of their TechPass account for any of the following reasons:
 
->**Important**
->- If your TechPass domain is **not** ```techpass.gov.sg```:
-> 1. [Create service request](https://go.gov.sg/techpass-sr) to remove the MFA of your TechPass profile.When the request is processed successfully, you will be able to register a new TechPass MFA profile when you try to sign in again.
-> 2. Complete the steps provided in this email.  
+- Changed mobile phone.
+- Lost mobile phone.
+- Deleted the Authenticator app from the mobile phone or unable to access the app on your phone.
 
+?> If you have a WOG account, you also need to [reset your MFA of your WOG account](reset-techpass-mfa-for-new-device).
 
-**Preqrequisites** 
 
-If you have changed the mobile phone and want to reset your TechPass MFA on the new mobile phone:
 
-- You must have the old mobile phone.
-- You must be able to authenticate your TechPass login using the old mobile phone.
+## Prerequisites
 
->**Note**
-> 
-> If you have lost your old mobile phone, formatted the old phone(factory reset),or unable to authenticate your log in using the old mobile phone, [Create service request](https://go.gov.sg/techpass-sr) to reset MFA for their TechPass profile.
+- You must have the old mobile phone to authenticate your TechPass login.
 
+?> If you have lost your old mobile phone, formatted the old phone(factory reset),or unable to authenticate your TechPass log in using the old mobile phone, contact [TechPass support team](https://go.gov.sg/techpass-sr).
```

Comment:
1. Put a space between **,** and **or**.
2. Is it log in or login?

Reply:
in this case , it should be login. thanks!

Reply:
not sure if i should use and or together. I have put a space and using only or.